bin/deploy.sh

```diff
@@ -23,19 +23,29 @@ yarn install --check-files --production=false
 # Download fonts from S3 bucket
 aws s3 cp s3://eds.ethoslabs.io/fonts.zip bin/fonts.zip
 if [[ ! -f "bin/fonts.zip" ]]; then
-  echo "Error: bin/fonts.zip does not exist."
+  echo "Error: \`bin/fonts.zip\` does not exist."
```

Comment:
This is interesting to me. Does it come out looking "code formatted" to stdout?

Reply:
no just comes out like this: \`wateverz\`

Reply:
gotcha